### DISCUSSION TITLE
Pending Timeout error

### DISCUSSION THREAD

**Discussion by aptx1231** [PaddlePaddle/PaddleSpatial#283] [BaiduKDDCup2022] [answered]
I encountered a pending timeout error on my submission on 2022-07-16 23:05 Beijing. The number of submissions should be refreshed at 2022-07-17 00:00:00 AOE time as agreed, but at 2022-07-17 00:00:00 Beijing, the daily quota has been refreshed to 3 times. If the total number of submissions is only 9, does it mean that the first 3 submissions need to be completed within 4 hours? In this case, we lose a chance to submit due to the pending timeout error.
So I would like to know if the total number of submissions is 9 or 12, and if it is 9, is that not fair enough for those in mainland China, as we must take advantage of these first 3 submissions between 2022-07-16 20:00-24:00.

**Comment by ramber1836** [comment, marked as answer]
> I encountered a pending timeout error on my submission on 2022-07-16 23:05 Beijing. The number of submissions should be refreshed at 2022-07-17 00:00:00 AOE time as agreed, but at 2022-07-17 00:00:00 Beijing, the daily quota has been refreshed to 3 times. If the total number of submissions is only 9, does it mean that the first 3 submissions need to be completed within 4 hours? In this case, we lose a chance to submit due to the pending timeout error.
> 
> So I would like to know if the total number of submissions is 9 or 12, and if it is 9, is that not fair enough for those in mainland China, as we must take advantage of these first 3 submissions between 2022-07-16 20:00-24:00.

We can re-evaluate those submissions encountering Pending Timeout Error.

**Reply by ytan1984** [reply]
> > I encountered a pending timeout error on my submission on 2022-07-16 23:05 Beijing. The number of submissions should be refreshed at 2022-07-17 00:00:00 AOE time as agreed, but at 2022-07-17 00:00:00 Beijing, the daily quota has been refreshed to 3 times. If the total number of submissions is only 9, does it mean that the first 3 submissions need to be completed within 4 hours? In this case, we lose a chance to submit due to the pending timeout error.
> > So I would like to know if the total number of submissions is 9 or 12, and if it is 9, is that not fair enough for those in mainland China, as we must take advantage of these first 3 submissions between 2022-07-16 20:00-24:00.
> 
> We can re-evaluate those submissions encountering Pending Timeout Error.

Then what do you plan to do for those who intended to leave the change to later of 16th July AOE time, but found the chance is reset ???

**Reply by ramber1836** [reply]
> > > I encountered a pending timeout error on my submission on 2022-07-16 23:05 Beijing. The number of submissions should be refreshed at 2022-07-17 00:00:00 AOE time as agreed, but at 2022-07-17 00:00:00 Beijing, the daily quota has been refreshed to 3 times. If the total number of submissions is only 9, does it mean that the first 3 submissions need to be completed within 4 hours? In this case, we lose a chance to submit due to the pending timeout error.
> > > So I would like to know if the total number of submissions is 9 or 12, and if it is 9, is that not fair enough for those in mainland China, as we must take advantage of these first 3 submissions between 2022-07-16 20:00-24:00.
> > 
> > 
> > We can re-evaluate those submissions encountering Pending Timeout Error.
> 
> Then what do you plan to do for those who intended to leave the change to later of 16th July AOE time, but found the chance is reset ???

They still have extra chances to submit their model from 04:00 to 20:00 on July 18 (AoE Time), right?

**Reply by aptx1231** [reply]
> We can re-evaluate those submissions encountering Pending Timeout Error.

But I've already resubmitted it myself (which should be a normal reaction after encountering an error report), taking up the next day's count.

**Reply by ramber1836** [reply]
> > We can re-evaluate those submissions encountering Pending Timeout Error.
> 
> But I've already resubmitted it myself (which should be a normal reaction after encountering an error report), taking up the next day's count.

As I said, you can take the quota from 04:00 to 20:00 on July 18 (AoE Time) as an extra chance for more submissions.

**Reply by ytan1984** [reply]
For myself, as long as the organizer is trying to make the competition open and fair, that will be fine and satisfied.

As @aptx1231 mentioned, the extra chance also bring other issues.
So maybe you could review the submit data, and give us a final decision.

**Reply by ramber1836** [reply]
> For myself, as long as the organizer is trying to make the competition open and fair, that will be fine and satisfied.
> 
> As @aptx1231 mentioned, the extra chance also bring other issues. So maybe you could review the submit data, and give us a final decision.

Fine, we will also take the number of submissions into account when making the final decision.

**Reply by fersebas** [reply]
Sorry, but I don't understand, the limit isn't it 11:59 PM on July 18 (AoE Time) as it says in the webpage?

**Reply by ramber1836** [reply]
The DDL for the submission in the 3rd phase is 11:59 PM on July 18 (AoE Time). 
 
The thing is that the strategy of counting and refreshing the quota for submissions is designed by the AiStudio platform, which counts the number of submissions according to the Beijing local time (UTC+8). 
So, one can make 12 submissions at most in the 3rd Phase, and the first three submissions were supposed to be made within 4 hours (00:00 AM to 03:59 AM on July 16 (AoE Time)), which is definitely too rushed for the competitors.  Sorry for such inconvenience. 
Subsequently, from 04:00 AM to 11:59 PM on July 18 (AoE Time) (or, 00:00 AM to 19:59 PM on July 19 (UTC+8 Time)), there is an extra quota for all competitors to make another 3 submissions within 20 hours. 

Again, apologies for such confusing settings.
